testsuite/tests/empty-ppc64le4.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+
+# testedfiles: helper.rb
+
+module Yast
```

Comment:
I'm missing a description of this test. Please add at least a comment what it tests and how. And what is the expected output.